src/utils.cpp

```diff
@@ -754,6 +755,9 @@ auto get_pkglist_base(const std::string_view& packages) noexcept -> std::vector<
         if (is_root_on_btrfs) {
             pkg_list.insert(pkg_list.cend(), {"snapper", "btrfs-assistant-git"});
         }
+        if (is_root_on_bcachefs) {
+            pkg_list.insert(pkg_list.cend(), {"bcachefs-tools"});
+        }
```

Comment:
that should be outside of server_mode check, so we have `bcachefs-tools` installed in both modes